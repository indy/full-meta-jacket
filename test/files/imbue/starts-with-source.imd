---
---



    (define x 42)
    (define y 100)
