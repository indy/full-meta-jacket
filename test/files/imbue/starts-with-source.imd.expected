<pre><code>(define x 42)
(define y 100)</code></pre>
